Cargo.toml

```diff
@@ -179,12 +218,6 @@ version = "7.3.1"
 [target."cfg(windows)".dependencies.ansi_term]
 version = "0.12.1"
 
-[dev-dependencies.assert_cmd]
-version = "2.0.16"
-
-[dev-dependencies.rusty-hook]
```

Comment:
I believe rusty-hook is needed as a dev dependency to run the git commit hook.
https://crates.io/crates/rusty-hook